api/client/samples/analysis_kits/stocks_to_use_price_model.py

```diff
@@ -0,0 +1,271 @@
+"""
+Sample Gro API Client script for regressing stocks-to-use ratios vs price
+
+This script creates a basic CME price valuation model using the US stocks-to-use ratio.
```

Comment:
Could you add a bit more descriptive information in the comments? Perhaps a sentence or two describing the method and then an indication of what the output of this will be.

Reply:
added a few sentences, PTAL